envision/server.py

```diff
@@ -318,6 +320,7 @@ async def on_message(self, message):
             if not prefix or event != "number":
                 continue
             frame_time = float(value)
+            break
```

Comment:
I remember doing this. This might be a simplified logic.
```suggestion
            if prefix and event == "number":
                frame_time = float(value)
                break
```